src/app/index.ts

```diff
@@ -433,16 +430,21 @@ function getDirectories(root) {
   });
 }
 
-function getFiles(root) {
-  return fs.readdirSync(root).filter(file => {
-    return !(fs.statSync(path.join(root, file)).isDirectory());
-  });
-}
-
 function updateHostNames(arr, key, newval) {
   let match = _.some(arr, _.method('match', key));
   if (match) {
     let index = _.indexOf(arr, key);
     arr.splice(index, 1, newval);
   }
+}
+
+function updateProjectNames(projectTemplate)
+{
+  if (projectTemplate == 'Manifest'){
+    projectTemplate = 'Office Add-in ' + projectTemplate + ' only';
+  }
+  else{
+    projectTemplate = 'Office Add-in using ' + projectTemplate + ' framework';

```

Comment:
Custom Functions is not a framework. Seems better to have a function that maps each projectTemplate string to a project display name string. Instead of updateProjectNames(projectTemplate) I would propose getProjectDisplayName(projectTemplate) and use a switch that just returns each string. Don't worry about trying to use projectTemple as part of the string formatting.  Even better would be to have this be data driven from a json file so adding a new one wouldn't require changing the function but just returning the mapping.